Task_TheFlexGame/index.css

```diff
@@ -1 +1,52 @@
-/* Start coding below for FLEX GAME TASK*/
+/* Start coding below for FLEX GAME TASK*/

```

Comment:
This line has an incorrect syntax. The asterisk should be inside the comment block.